be/src/util/mem_info.cpp

```diff
@@ -59,6 +60,11 @@ void MemInfo::init() {
         if (result == StringParser::PARSE_SUCCESS) {
             // Entries in /proc/meminfo are in KB.
             _s_physical_mem = mem_total_kb * 1024L;
+            int64_t cgroup_mem_limit;
```

Comment:
```suggestion
            int64_t cgroup_mem_limit = 0;
```
better to give a default value

Reply:
seems better to move this out of while block.